datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -39,24 +39,12 @@ def backend_normalize_metric_name(metric_name):
 
 
 def check_tag_names(metric, tags):
-    forbidden_tags = [
-        'cluster_name',
-        'clustername',
-        'cluster',
-        'clusterid',
-        'cluster_id',
-        'env',
-        'host_name',
-        'hostname',
-        'host',
-        'service',
-        'version',
-    ]
-
     if not os.environ.get('DDEV_SKIP_GENERIC_TAGS_CHECK'):
+        from datadog_checks.base.utils.tagging import GENERIC_TAGS
```

Comment:
```suggestion
        try:
            from datadog_checks.base.utils.tagging import GENERIC_TAGS
        except ImportError:
            GENERIC_TAGS = []
```
Base check is not a requirement of ddev